config/nodeMailer.js

```diff
@@ -0,0 +1,49 @@
+var CONSTANT = require('../config/constant');
```

Comment:
It does not look like this file is under version control. It looks like it contains a Gmail account username and password which is probably why. A nice solution here might be to use environment variables. Test and dev environment would have a dev Gmail account setup to use. Production would have the real account, and those credentials would only be stored on that machine.